rootfs/bin/boot

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+echo "Powered by $POWERED_BY" > /usr/share/nginx/html/index.html
```

Comment:
AFAIK html being served by nginx won't recognize the environment variable and just serve the literal string "$POWERED_BY".